servatrice/src/servatrice.cpp

```diff
@@ -462,7 +462,12 @@ void Servatrice::scheduleShutdown(const QString &reason, int minutes)
     if (minutes > 0) {
         shutdownTimer = new QTimer;
         connect(shutdownTimer, SIGNAL(timeout()), this, SLOT(shutdownTimeout()));
-        shutdownTimer->start(60000);
+        if (minutes <= 5 || isFirstShutdownMessage) {
+            shutdownTimer->start(60000);
+            isFirstShutdownMessage = false;
+        }
+        else
+            shutdownTimer->start((shutdownMinutes / 2) * 60 * 1000);
```

Comment:
The logic of this doesn't make sense to me... Wouldn't this trigger every minute still?